### PR TITLE
Readme - Localization to pt-PT

### PR DESCRIPTION
**Pull request by gablm**
Basically what the title says, and the addition of a link to the new Readme from the Readmes in other languages. 

Issue/PR #300 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ##### GI Nahida Background Credit: [Rafa on Pixiv](https://www.pixiv.net/en/users/3970196)
 
-[![jp](https://img.shields.io/badge/README-jp-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [![id](https://img.shields.io/badge/README-id-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md)
+[![jp](https://img.shields.io/badge/README-jp-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [![id](https://img.shields.io/badge/README-id-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md) [![pt](https://img.shields.io/badge/lang-pt-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.pt-pt.md)
```
Comment (bagusnl) [inline README.md:8]: Please update this badges with the new text 
```
[![jp](https://img.shields.io/badge/README-jp-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [![id](https://img.shields.io/badge/README-id-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md)
```

**Review by bagusnl — CHANGES_REQUESTED** [review]
Everything is all good except the badges in main readme file that needs a minor change

File: README.md (hunk 1 above)
Comment (gablm) [reply README.md]: Like this?
` [jp](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [id](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md) [pt](https://github.com/neon-nyan/Collapse/blob/main/README.pt-pt.md) `

Comment (bagusnl) [reply README.md]: Yep, just lang to README in the main readme badges

Comment (gablm) [reply README.md]: I think it's fixed now

**Review by bagusnl — APPROVED** [review]
All looks good on me, merging to main.
Thank you for translating our ReadMe and congrats on having PR/Issue no 300 🎉

**Pull request merged**